.github/workflows/ec2-integration-test.yml

```diff
@@ -5,9 +5,8 @@ name: Reusable EC2 Integration Test
 
 env:
   PRIVATE_KEY: ${{ secrets.AWS_PRIVATE_KEY  }}
-  TERRAFORM_AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
```

Comment:
This comes from inputs since the creds and bucket will not be the same in itar as commercial.